pkgs/misc/emulators/fceux/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fceux";
-  version = "2.5.0";
+  version = "2.6.0";
 
   src = fetchFromGitHub {
     owner = "TASVideos";
```

Comment:
Looks like it redirects to "TASEmulators" now?

Reply:
I changed it to `TASEmulators`. They seem to have split emulator related projects into a separate organisation. Thanks for noticing that!